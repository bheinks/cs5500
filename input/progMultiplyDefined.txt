(*
	Program name is multiply defined identifier
*)

program p;

var x : integer;
    p : boolean;
    z : char;

begin
  write(5)
end.

