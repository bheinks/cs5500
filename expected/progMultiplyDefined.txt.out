

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding x to symbol table with type INTEGER

+++ Adding p to symbol table with type BOOLEAN
Line 8: Multiply defined identifier
